src/repo.rs

```diff
@@ -59,6 +59,24 @@ impl Repository {
         }
     }
 
+    /// Find and open a repository, with additional options.
```

Comment:
Because flags aren't exactly namespaced just yet, could you add some examples here of what values `flags` can take on?